src/components/Preview.jsx

```diff
@@ -32,6 +36,7 @@ class Preview extends React.Component {
         propagateErrorsToParent: isLivePreview,
         breakLoops: isLivePreview,
         nonBlockingAlertsAndPrompts: isLivePreview,
+        refreshTimestamp: timestamp,
```

Comment:
I think we’ll want to only pass this in if it’s `isLivePreview` (the pop-out version should generally be as “pure” as possible).